content/community/engineering_guidelines.md

```diff
@@ -50,6 +50,12 @@ Using [zenhub.io](https://zenhub.io) allows us to link PRs to issues without nec
 
 Pretty self-explanatory: no self-merged PRs without prior approval from maintainers. Reviews are everyone's responsibility and should focus on tests and documentation as much as implementation.
 
+However, it should taken care that the PR review process does not take too long as it is easy to get lost in the reviews:
+
+* A PR which is marked as 'done' (i.e. not marked as 'work-in-progress') should be merged or rejected with 3 business days.
+* There should be at most 2 review turnarounds (i.e. reviewer requests changes - author makes changes - reviewer request final changes - author makes changes - reviewer accepts / rejects).
+* A reviewer should be actively selected by the author immediately after creating the PR, e.g. asking on IRC or mailing list.
```

Comment:
Doesn't jimmibot already assign reviewers to the PRs, perhaps a model if you're assigned to review the PR you should either commit to review it within the allotted time or take your self off the reviewers list (I think that can be done) and notify the contributor?